.eslintrc.js

```diff
@@ -4,6 +4,8 @@ const frontPaths = [
   'packages/**/ee/admin/**/*.js',
   'packages/core/helper-plugin/**/*.js',
   'packages/**/tests/front/**/*.js',
+  'packages/core/telemetry/admin/index.js',
+  'packages/core/telemetry/admin/src/ampli/index.js',
```

Comment:
you add this path here but the src/ampli folder is ignored in .eslintignore

Reply:
For some reason if I don't include these lines here, webpack does not want to build admin telemetry when I test it locally. Any ideas why this could happen?